src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -58,20 +62,20 @@ if [ "$BASE_URL" == "" ]; then
     BASE_URL=$(./bin/ceph mgr services | jq -r .dashboard)
 fi
 
-export BASE_URL
-
 cd $DASH_DIR/frontend
 jq .[].target=\"$BASE_URL\" proxy.conf.json.sample > proxy.conf.json
 
 [[ "$(command -v npm)" == '' ]] && . ${FULL_PATH_BUILD_DIR}/src/pybind/mgr/dashboard/node-env/bin/activate
 
 if [ "$DEVICE" == "chrome" ]; then
+    export BASE_URL E2E_LOGIN_USER E2E_LOGIN_PWD
     npm run e2e:ci || stop 1
     stop 0
 elif [ "$DEVICE" == "docker" ]; then
     failed=0
     docker run -d -v $(pwd):/workdir --net=host --name angular-e2e-container rogargon/angular-e2e || failed=1
-    docker exec -e BASE_URL=$BASE_URL angular-e2e-container npm run e2e:ci || failed=1
+    docker exec -e BASE_URL="$BASE_URL" -e E2E_LOGIN_USER="$E2E_LOGIN_USER" -e E2E_LOGIN_PWD="$E2E_LOGIN_USER" \
```

Comment:
Given we are currently passing 3 envs (and especially a password), why not using an 'env' file instead?

```suggestion
    docker exec --env-file env-file \
```
Or, if we don't want to take care of that file:
```suggestion
    env=$(cat <<-EOF
BASE_URL="$BASE_URL"
E2E_LOGIN_USER="$E2E_LOGIN_USER"
E2E_LOGIN_PWD="$E2E_LOGIN_USER"
EOF
)
    docker exec --env-file <(echo "$env") ...
```

Reply:
@epuertat @LenzGr Unfortunately --env-file only works with ```docker run```.
But I realizaed that the docker image used here is outdated and no longer works for us,
So following the suggestion I made a refactor using ```docker run```
and docker image ```docker.io/rhcsdashboard/e2e``` as a workaround.
We can agree on publish a more suitable docker image like
```docker.io/ceph-dashboard/e2e```
@tspmelo @callithea Thoughts?

Reply:
Sounds good to me!

Reply:
I like the idea, but would like to suggest to publish this image under the already existing [Ceph organization on DockerHub](https://hub.docker.com/u/ceph), to avoid confusion.

Reply:
Should we create a tracker issue to keep this in mind if we want to merge this PR as it is (for now)?